kalite/distributed/features/steps/superuser_create.py

```diff
@@ -0,0 +1,145 @@
+from behave import *
+from selenium.common.exceptions import NoSuchElementException
+
+from django.core.urlresolvers import reverse
+
+from kalite.testing.behave_helpers import *
+
+from kalite.testing.mixins.facility_mixins import CreateFacilityMixin, CreateStudentMixin
+
+
+from django.contrib.auth.models import User
+
+@given("there is superuser")
+def step_impl(context):
+    if User.objects.exists():
+        pass
+    else:
+        User.objects.create_superuser(username='superusername', password='superpassword', email='super@email.com')
+
+@then("there should be no modal displayed")
+def step_impl(context):
+    assert not find_id_with_wait(context, "superusercreate-container")
```

Comment:
I also recommend pulling the id `"superusercreate-container"` into a variable, for easy changing.